packages/airnode-deployer/src/utils/infrastructure.ts

```diff
@@ -17,3 +17,8 @@ export function formatTerraformArguments(args: CommandArg[]) {
     })
     .map((arg) => `-${arg}`);
 }
+
+/**
+ * Checks if the environment is a GCP or AWS cloud function
+ */
+export const isCloudFunction = () => process.env.LAMBDA_TASK_ROOT || process.env.FUNCTION_TARGET;
```

Comment:
I wonder how reliable this check is. Will these always be present?

Reply:
According to the Google and AWS docs, yes:
[AWS](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
[GCP](https://cloud.google.com/functions/docs/configuring/env-var)
A library to do this would be ideal, but checking for those ENVs is quick, should be reliable and doesn't require any additional dependencies.